includes/admin/tools.php

```diff
@@ -1377,7 +1377,8 @@ function edd_tools_sysinfo_get() {
 			continue;
 
 		$update = ( array_key_exists( $plugin_path, $updates ) ) ? ' (needs update - ' . $updates[$plugin_path]->update->new_version . ')' : '';
-		$return .= $plugin['Name'] . ': ' . $plugin['Version'] . $update . "\n";
+		$plugin_url = isset( $plugin['PluginURI'] ) && ! empty( $plugin['PluginURI'] ) ? $plugin['PluginURI'] : $plugin['AuthorURI'];
```

Comment:
This is not happening for me, at least not with the plugins I have active, but what if a plugin header is missing both the `PluginURI` and the `AuthorURI`? I'm wondering about doing something like this instead:

```php
		$plugin_url = '';
		if ( ! empty( $plugin['PluginURI'] ) ) {
			$plugin_url = $plugin['PluginURI'];
		} elseif ( ! empty( $plugin['AuthorURI'] ) ) {
			$plugin_url = $plugin['AuthorURI'];
		}
```

Note that `empty()` will also satisfy `isset()` so you don't need both.

Reply:
Updated!